### PR TITLE
division

### PR DESCRIPTION
**Pull request by alexandruradovici**

### CODE REVIEW

**Comment by etrvic** [comment]
what happens when you divide by 0?

File: main_test.ts [hunk 1]
```diff
@@ -12,3 +12,11 @@ Deno.test(function subTest() {
 Deno.test(function sub2Test() {
   assertEquals(sub(2, 3), -1);
 });
+
+Deno.test(function divTest() {
+  assertEquals(div(3, 2), 1.5);
+});
+
+Deno.test(function intDivTest() {
```
Comment (etrvic) [inline main_test.ts:20]: division by 0 not tested

**Review by etrvic — CHANGES_REQUESTED** [review]
division by 0

File: main_test.ts (hunk 1 above)
Comment (alexandruradovici) [reply main_test.ts:20]: Sure, adding tests.

Comment (alexandruradovici) [reply main_test.ts:20]: Done, please review.

**Review by etrvic — APPROVED** [review]

**Review by etrvic — APPROVED** [review]
done

**Pull request merged**